tests/expr_and_series/is_null_test.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any
+
+import narwhals.stable.v1 as nw
+from tests.utils import compare_dicts
+
+
+def test_null(constructor: Any) -> None:
+    data_na = {"a": [None, 3, 2], "z": [7.0, None, None]}
+    expected = {"a": [True, False, False], "z": [True, False, False]}
+    df = nw.from_native(constructor(data_na))
+    result = df.select(nw.col("a").is_null(), ~nw.col("z").is_null())
```

Comment:
Apparently, this is the only place in which we also test `__invert__`, that's why the implementation ended up in the same pr